### PR TITLE
updated event status 

### PR DESCRIPTION
**Pull request by yasster**
#33 This is working although, it goes through the logs twice, as that can be a performance issue!

Thanks to @grazietta, I'll continue with weekly and monthly metrics using Moment.js

### CODE REVIEW

**Comment by alireza-a** [comment]
update the name and squash your commits

File: src/status/status.factory.js [hunk 1]
```diff
@@ -34,19 +51,35 @@ module.exports = (logProvider, logger) => {
     return memeberStatus;
   }
 
+  function eventMemberStatus(events) {
```
Comment (grazietta) [inline src/status/status.factory.js]: can you rename this to something like 'allTimeEventStatus' to keep it consistent with 'allTimeMemberStatus' or you could rename them both to something else that is more explicit

File: src/status/status.spec.js [hunk 2]
```diff
@@ -40,12 +40,31 @@ describe('status.fatory', function() {
...
+      .then(({ eventStatus }) => {
+        expected.forEach(expectedStatus => {
+          const actualStatus = eventStatus.get(expectedStatus.event_id);
+          expect(actualStatus).to.deep.equal(expectedStatus);
+        });
+        done();
+      })
+      .catch(err => done(err));
+  });
+
```
Comment (grazietta) [inline src/status/status.spec.js]: remove this space here

File: src/status/status.factory.js [hunk 3]
```diff
@@ -34,19 +51,35 @@ module.exports = (logProvider, logger) => {
...
+    const eventStatus = initializeEventStatus(eventIds);
+    events.forEach(({ event_id, action }) => {
+      const status = eventStatus.get(event_id);
+      if (action === 'Participated') status.participated += 1;
+      if (action === 'Notified') status.notified += 1;
+    });
+    return eventStatus;
+  }
+
+
```
Comment (grazietta) [inline src/status/status.factory.js]: remove the extra new line

**Pull request merged**